standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStore.java

```diff
@@ -571,9 +571,12 @@ public void init() throws MetaException {
       listeners = MetaStoreServerUtils.getMetaStoreListeners(MetaStoreEventListener.class, conf,
           MetastoreConf.getVar(conf, ConfVars.EVENT_LISTENERS));
       listeners.add(new SessionPropertiesListener(conf));
-      transactionalListeners = MetaStoreServerUtils.getMetaStoreListeners(TransactionalMetaStoreEventListener.class,
-          conf, MetastoreConf.getVar(conf, ConfVars.TRANSACTIONAL_EVENT_LISTENERS));
-      transactionalListeners.add(new AcidEventListener(conf));
+      transactionalListeners = new ArrayList() {{
```

Comment:
Hey @pkumarsinha thanks for the PR -- along the same lines as @aasha , would it make sense to have have some checks in place ensuring some order depending on the Listener type instead of just relying on the user (who will highly likely ignore the conf comment you added above)

Reply:
@pgaref  Thanks for the review. 
Currently, there are just two impl for transnational event listener in Hive . Out of that, AcidEventLisner is hard-coded to be included. The other one, DbNotificationListener is the one which will ideally end up getting configured if user choose  to do so. So the idea was,  unless a user writes a custom transnational event listener, this config can have at most one value as of now. Let me know your opinion with this context.

Reply:
Got it, makes sense!